CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [2.13.0] - 2022-06-21 - Yeonpyeongdo (연평도, 延坪島)
+
+### Added
+- EXtended headers usage clarification (#204)
```

Comment:
s/X/x/

Reply:
thought that "eXtended header" is good naming. but it looks strange with the first capitalized letter, agree

fixed